### PR TITLE
BREAKING: remove std/testing/runner.ts, use deno test for std/ tests

### PR DESCRIPTION
**Pull request by bartlomieju**
This PR removes `std/testing/runner.ts` and migrates `std/` tests to use `deno test` in an effort to unify testing.

### CODE REVIEW

File: std/encoding/binary_test.ts [hunk 1]
```diff
@@ -160,5 +160,3 @@ Deno.test(async function testWriteVarnumLittleEndian(): Promise<void> {
   await buff.read(data);
   assertEquals(data, new Uint8Array([0x01, 0x02, 0x03, 0x04]));
 });
-
-Deno.runTests();
```
Comment (bartlomieju) [inline std/encoding/binary_test.ts]: This is very suspicious, might have been that some of the tests were run twice... We need to guard against multiple calls to `runTests`

**Comment by ry** [comment]
@bartlomieju I'm a bit confused with these PRs, can you merge this with master? Then I will take a look.

**Comment by bartlomieju** [comment]
> @bartlomieju I'm a bit confused with these PRs, can you merge this with master? Then I will take a look.

Yeah, something went wrong during merge yesterday

**Comment by bartlomieju** [comment]
Should be landed after https://github.com/denoland/deno/pull/4392

**Pull request merged**